lib/contracts/modules.rb

```diff
@@ -10,8 +10,8 @@ def self.extended(base)
 
     def self.common(base)
       return unless base.instance_of?(Module)
-      base.extend(MethodDecorators)
-      Eigenclass.lift(base)
+      #base.extend(MethodDecorators)
```

Comment:
So this one became a no-op. Effectively means, that `Contracts::Modules` can be safely removed.